README.md

```diff
@@ -361,6 +361,34 @@ Class2       0            10
 >>> auc_trp[2]
 0.29166666666666663
 ```
+### Precision curve
+
+`PCurve`, added in `version 4.6`, is devised to compute the Precision curve in which the Y axis represents the Precision, and the X axis represents the discrimination thresholds applied to a classifier. Thus, the ideal point is located at the top left of the curve, and a larger area under the curve represents better performance. Precision curve is a graphical representation of binary classifiers' performance. In PyCM, `PCurve` binarizes the output based on the "One vs. Rest" strategy to provide an extension of this curve for multi-class classifiers. Getting the actual labels vector, the target probability estimates of the positive classes, and the list of ordered labels of classes, this method is able to compute and plot Precision per different discrimination thresholds and compute the area under the curve.
```

Comment:
We shouldn't talk about the ideal point for this curve ig; I don't think top-left is the ideal point, nor the area under the curve represents anything meaningful (robustness to threshold or something like this at most if it backs up with literature)